test/run.d

```diff
@@ -67,6 +67,7 @@ Examples:
     ./run.d runnable/template2962.d                              # runs a specific tests
     ./run.d runnable/template2962.d fail_compilation/fail282.d   # runs multiple specific tests
     ./run.d fail_compilation                                     # runs all tests in fail_compilation
+    ./run.d unit_tests                                           # runs all unit tests
```

Comment:
Perhaps we should not document it since the `-u` flag exists as well.

Reply:
Hmm, but it's documented in the [`README.md`](https://github.com/dlang/dmd/blob/master/test/README.md#running-the-unit-tests):

> The unit tests will automatically run when all tests are run using `./run.d` or
`make`. To only run the unit tests the `./run.d unit_tests` command can be used.
For a more finer grain control over the unit tests the `./run.d -u` command can
be used:

I guess as long as we're consistent I'm fine either way. Other thoughts.

Btw currently there's no easy way in to run just the integration tests (i.e. compilable, runnable, fail_compilation). Maybe it's worth adding `tests` as a shortcut?

Reply:
> Hmm, but it's documented in the README.md

Yes, but we can remove that as well. It's a bit inconsistent to have `unit_tests` and the `-u` flag.

> Btw currently there's no easy way in to run just the integration tests (i.e. compilable, runnable, fail_compilation). Maybe it's worth adding `tests` as a shortcut?

Perhaps.